test/markdownlint-cli2-test-cases.js

```diff
@@ -73,7 +78,11 @@ const testCases =
               "exitCode": child.exitCode,
               "stdout": sanitize(child.stdout),
               "stderr": sanitize(child.stderr),
-              "formatterCodeQuality": sanitize(formatterOutputCodeQuality),
+              "formatterCodeQuality":
+                sanitize(
```

Comment:
Line length was too long to use a single line?

Reply:
Yup, this was 90 characters all on one line, which exceeds the 80-character limit.